utils/Command/MissingInSetCommand.php

```diff
@@ -34,6 +35,8 @@ final class MissingInSetCommand extends Command
         FinalizeClassesWithoutChildrenCollectorRector::class,
         // changes behavior, should be applied on purpose regardless PHP 7.3 level
         JsonThrowOnErrorRector::class,
+        // in confront with sub type safe belt detection on RemoveUseless*TagRector
+        RemoveNullTagValueNodeRector::class,
```

Comment:
this registered in `MissingInSetCommand` on purpose to avoid use null tag on purpose, which already on safe belt on `RemoveUseless*TagRector`